src/platform/plugins/shared/data/common/search/expressions/esql.ts

```diff
@@ -272,7 +272,7 @@ export const getEsqlFn = ({ getStartDependencies }: EsqlFnArguments) => {
             IKibanaSearchResponse<ESQLSearchResponse>
           >(
             { params: { ...params, dropNullColumns: true } },
-            { abortSignal, strategy: ESQL_ASYNC_SEARCH_STRATEGY }
+            { abortSignal, strategy: ESQL_ASYNC_SEARCH_STRATEGY, sessionId: getSearchSessionId() }
```

Comment:
Does this need to be `searchSessionId` as well? (I'm looking at src/platform/plugins/shared/data/public/search/expressions/esaggs.ts)

Reply:
You're right, I'm not sure why I thought otherwise